### ISSUE TITLE
Improve display of internal transactions on /tx

### ISSUE THREAD

**Issue by poplexity** [telosnetwork/teloscan#93]
## Overview
It would improve user experience if the internal transactions in the transaction view got the same UI treatment as the Logs tab.

## Acceptance criteria
- internal transactions, eg. [here](https://www.teloscan.io/tx/0xcfb64bca2f55829f899a181e5b878f2cb5b43cd315087e5d8b664fbd43bc67c4#internal), should have the same appearance and view mode toggle as the Logs tab

## Technical considerations
N/A

**Comment by EJSG** [comment]
Blocked by #107 as this tab does not currently load at all

**Comment by tomtomcrypto** [comment]
Would also be nice to have the Internal tx tab that etherscan has on the account page.
